index.html

```diff
@@ -30,7 +30,36 @@
     <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
     <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <!-- For Lightbox Effect-->
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <style type="text/css">
+        a.fancybox img {
+            border: none;
+            -o-transform: scale(1, 1);
+            -ms-transform: scale(1, 1);
+            -moz-transform: scale(1, 1);
```

Comment:
-moz-, -ms-, and -o- are for very old versions of their respective browsers. you dont need to include them at all.